src/Test/TestAIBehavior.java

```diff
@@ -165,15 +167,15 @@ private boolean testClearAttackRoute(AIMaker ai)
     Unit iRight = addUnit(testMap, testCo1, "Infantry", facPos.right());
 
     // The infs all want to cap the fac, but can't because it is occupied. Gotta let the AA through.
-    testCo1.initTurn(testMap);
-    testCo1.money = 0; // No production needed for this test.
+    testCo1.army.initTurn(testMap);
+    testCo1.army.money = 0; // No production needed for this test.
```

Comment:
Why not store the army as part of the test setup instead of calling `testCo1.army` every time?

Reply:
Apathy.